### PR TITLE
Adding @kisantia as a codeowner for dacpac and schema compare extensions

### PR DESCRIPTION
**Pull request by kisantia**
Adding codeowners for the dacpac, schema compare, and sql database projects extensions

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,4 +6,5 @@
 /src/extensions/resource-deployment/ @ranasaria
 /src/extensions/arc/ @ranasaria
 /src/extensions/azdata/ @ranasaria
-
+/extensions/dacpac/ @kisantia
```
Comment (kisantia) [inline .github/CODEOWNERS:9]: is this the right path to use? I see Arvind has /src/ at the start of the paths he added, but the extensions aren't in the src folder

Comment (alanrenmsft) [inline .github/CODEOWNERS:9]: lol, do you mind fix the path for the entries Arvind added for him?

Comment (alanrenmsft) [reply .github/CODEOWNERS:9]: yours looks correct, but I would always test it before sending the PR. @ranasaria FYI.

**Review by alanrenmsft — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (alanrenmsft) [reply .github/CODEOWNERS:9]: you can test by creating a child branch from this one and then ask someone else to create a PR modifying the files in the folder you care about and see whether you are tagged automatically

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -3,7 +3,8 @@
 # Syntax can be found here: https://docs.github.com/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 /src/sql/*.d.ts @alanrenmsft @Charles-Gagnon @ranasaria
-/src/extensions/resource-deployment/ @ranasaria
-/src/extensions/arc/ @ranasaria
-/src/extensions/azdata/ @ranasaria
-
+/extensions/resource-deployment/ @ranasaria
```
Comment (alanrenmsft) [inline .github/CODEOWNERS:6]: thank you for fixing these.

File: .github/CODEOWNERS (hunk 1 above)
Comment (kisantia) [reply .github/CODEOWNERS:9]: @Benjn helped confirm that this worked :)

**Review by Benjin — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**